conf_files/huntsman.yaml

```
@@ -30,6 +30,7 @@ directories:
     resources: POCS/resources/
     targets: /var/huntsman/huntsman-pocs/conf_files
     webcam: webcams
+    sshfs_mount: /home/huntsman/Huntsmans-Pro
```

I'm not actually sure if this is true or not.

```suggestion
    sshfs_mount: /home/huntsman/Huntsman-Pro
```

---

Why not just use `images`?

---

Nice suggestion. I was worried that changing 'images' would impact other parts of the code - if it is only used here then that would be a good way to go.

---

I used "Huntsmans-Pro" for the mount point  because apparently that is its name (according to the google doc with the IP addresses)